README.md

```diff
@@ -1,141 +1,146 @@
+**Redistributing and Unlocking Juror Stake**
 
-If there is not enough active jurors within the court, we can select jurors from courts from higher levels in the court hierarchy, eventually pulling in every active juror in the supreme court to produce a decision which can no longer be appealed. In practice we expect the cost to escalate a dispute to that level would be prohibitive in all but the most extreme cases, but it is important that the option to escalate disputes is available.
-# 4. Aragon Network Token
-Agreements require agents to lock up collateral, in some cases for extended periods of time. If a highly volatile asset like ETH is used, parties of the agreement are exposing themselves to the volatility risk of the collateralized asset. On the other hand if a stablecoin like DAI is used both parties face an opportunity cost of holding an asset that has no upside potential. The ideal currency for collateralizing agreements would provide competitive risk adjusted returns for holders and frequently split to ensure it can still be used as a stable unit of account. It would also retain some of its utility while locked up.
+After a final ruling has been decided, all the adjudication rounds in the dispute can be settled. Jurors that didn't vote for the final ruling will lose the tokens that they had at stake. All the aggregated penalties and the juror fees in a round will be distributed proportionally among the jurors that voted for the final ruling option.
 
-To optimize ANT for usage as a reserve currency for agreements and minimize the opportunity cost of participating in governance of the Aragon Network, we propose a monetary policy that programmatically manages both the supply of ANT and a stability reserve based on a price target. In addition, to encourage participation and long run growth of the network we incorporate a participation rate target, inspired by [Livepeer](https://github.com/livepeer/wiki/blob/master/WHITEPAPER.md), that diverts a portion of inflation to reward specific groups of users who actively contribute to growing the value of the network. Using inflation to reward positive contributions to the network is similar in principle to the coinbase rewards in Bitcoin and Ethereum.
+In case that no juror in a round voted for the final ruling, the juror tokens will be burned and the collected juror fees for the round will be refunded to the creator of the dispute (in case that it was the first round) or to the appealing party that sided with the final ruling.
```

Comment:
I see the phrases "burned" and "slashed", what would you say are the differences between these terms as used here?

Reply:
Should just be slashed here.